posthog/test/test_client.py

```diff
@@ -494,6 +506,22 @@ def test_enabled_to_disabled(self):
 
         self.assertEqual(msg, "disabled")
 
+    def test_geoip_disable_default_on_events(self):
+        client = Client(FAKE_TEST_API_KEY, on_error=self.set_fail, geoip_disable=True)
+        _, capture_msg = client.capture("distinct_id", "python test event")
+        client.flush()
+        self.assertEqual(capture_msg["properties"]["$geoip_disable"], True)
+
+        _, identify_msg = client.identify("distinct_id", {"trait": "value"})
+        client.flush()
+        self.assertEqual(identify_msg["properties"]["$geoip_disable"], True)
+
+    def test_geoip_disable_method_overrides_init_on_events(self):
+        client = Client(FAKE_TEST_API_KEY, on_error=self.set_fail, geoip_disable=True)
+        _, msg = client.capture("distinct_id", "python test event", geoip_disable=False)
+        client.flush()
+        self.assertTrue("$geoip_disable" not in msg["properties"])
```

Comment:
we need tests for the flag functions too!

And one for `_active_feature_flag` on capture (i.e. when all flags are attached to an event)

Reply:
oh and one more case in this current test here where we initialise with False, pass in geoip_disable=True and see it in properties.

Reply:
> we need tests for the flag functions too!

all of them?! 😱

Reply:
smallest possible that covers all code paths, so nope probably not all, anything that goes to `get_decide` should work